airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeSQLNameTransformer.java

```diff
@@ -17,4 +17,7 @@ public String getStageName(String schemaName, String outputTableName) {
     return schemaName.concat(outputTableName).replaceAll("-", "_").toUpperCase();
   }
 
+  public String getStagingPath(String schemaName, String tableName, String currentSyncPath) {
+    return getStageName(schemaName,tableName).concat("/staged/").concat(currentSyncPath).toUpperCase();
```

Comment:
nitpick: why not use `+` instead of `.concat`?

Reply:
> nitpick: why not use `+` instead of `.concat`?

replaced with +